components/brave_wallet/common/eth_sign_typed_data_helper.cc

```diff
@@ -268,8 +268,9 @@ EthSignTypedDataHelper::EncodeField(const std::string_view type,
   }
 
   if (type.starts_with("bytes")) {
-    unsigned num_bits;
-    if (!base::StringToUint(type.substr(5), &num_bits) || num_bits > 32) {
+    unsigned num_bytes;
+    if (!base::StringToUint(type.substr(5), &num_bytes) || num_bytes == 0 ||
```

Comment:
Dumb question, I see we're using `&num_bytes` as an out-param but is `StringToUint` _guaranteed_ to write 0 to it?

If we can, we should just value-init here with `unsigned num_bytes = 0`.

Reply:
That's a good question. I just took a look at the test cases for this function and it does seem so: https://source.chromium.org/chromium/chromium/src/+/main:base/strings/string_number_conversions_unittest.cc;l=123

I'll see if I can value-init here as well.